SDK/simVis/BathymetryGenerator.vert.glsl

```diff
@@ -1,7 +1,7 @@
-#version 140
+#version $GLSL_VERSION_STR
 
 #pragma vp_entryPoint simVis_BathymetryGenerator_vertex
-#pragma vp_location vertex_clip
+#pragma vp_location vertex_view
```

Comment:
I am seeing a jittering in the ocean near the Bathymetry Generator data so I agree something is up.  But this change as-is completely breaks the Bathymetry Generator; Ocean is no longer seen where there is no elevation loaded, and that's a hard requirement.